spec/ataru/fixtures/db/browser_test_db.clj

```diff
@@ -91,6 +91,34 @@
                                          :params     {},
                                          :fieldType  "textField"}]})
 
+(def hakija-hakukohteen-hakuaika-test-form {:id 6
+                                       :key "hakija-hakukohteen-hakuaika-test-form"
```

Comment:
Map keys not aligned, indentation a bit messed up. You could also put the map on a new line completely to avoid  a large empty space on the left :)